modal/_functions.py

```diff
@@ -1477,6 +1480,11 @@ async def get_web_url(self) -> Optional[str]:
         """URL of a Function running as a web endpoint."""
         return self._web_url
 
+    @live_method
+    async def _experimental_get_flash_urls(self) -> Optional[list[str]]:
+        """URL of the flash service for the function."""
+        return self._experimental_flash_urls or None
```

Comment:
For the type checker:

```suggestion
        return list(self._experimental_flash_urls) or None
```